app/cleaning.py

```diff
@@ -0,0 +1,63 @@
+import json
+import pandas as pd
+import re 
+import ast
+import datetime
+
```

Comment:
great work - if you can add some comments in the code to explain what is going on and add doc string to function.